mongodb/src/main/scala/akka/stream/alpakka/mongodb/scaladsl/MongoSink.scala

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.mongodb.scaladsl
+
+import akka.stream.scaladsl.{Flow, Keep, Sink}
+import akka.{Done, NotUsed}
+import org.mongodb.scala.bson.conversions.Bson
+import org.mongodb.scala.model.UpdateOptions
+import org.mongodb.scala.result.{DeleteResult, UpdateResult}
+import org.mongodb.scala.{Completed, Document, MongoCollection}
+
+import scala.concurrent.Future
+
+object MongoSink {
+
+  /**
+   * A [[Sink]] that will insert documents into a collection.
+   * @param parallelism number of documents to insert in parallel.
+   * @param collection mongo db collection to insert to.
+   */
+  def insertOne(parallelism: Int, collection: MongoCollection[Document]): Sink[Document, Future[Done]] =
+    insertOneFlow(parallelism, collection).toMat(Sink.ignore)(Keep.right)
+
+  /**
+   * A [[Flow]] that will insert documents into a collection.
+   * @param parallelism number of documents to insert in parallel.
+   * @param collection mongo db collection to insert to.
+   */
+  def insertOneFlow(parallelism: Int, collection: MongoCollection[Document]): Flow[Document, Completed, NotUsed] =
```

Comment:
Make a `MongoFlow` and put these in that without prefix!

Reply:
Great idea.